.github/workflows/interpreter.yml

```
@@ -0,0 +1,64 @@
+name: Interpreter Test
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  SPEC_SPLIT_DOTS: 160
+
+jobs:
+  test-interpreter_spec:
```

Naming consistency-wise:

```suggestion
  test-interpreter-spec:
```

---

The spec suite that's being run is called `interpreter_spec`.